.env.sample

```diff
@@ -1,5 +1,5 @@
-export PGPORT=xxxx
-export PGHOST=xxx
-export PGPASSWORD=xxx
-export PGUSER=xxx
-export PGDATABASE=xxx
\ No newline at end of file
+PGPORT=xxxx
+PGHOST=xxx
+PGPASSWORD=xxx
+PGUSER=xxx
+PGDATABASE=xxx
```

Comment:
Without `export` these env variables didn't work for me in local development when I do `source .env`

Reply:
It doesn't matter since nothing reads from `.env.sample`.